src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -194,11 +196,109 @@ module OfficeInteropExtensions =
                                     |> Option.defaultValue ""
                                 )
                             )
+
                         ArcTable.fromStringSeqs(excelTable.name, headers, bodyRows)
                     )
                     return inMemoryTable
             }
 
+        /// <summary>
+        /// Checks whether the string seqs are part of a valid arc table or not and returns potential errors and index of them in the annotation table
+        /// </summary>
+        /// <param name="headers"></param>
+        /// <param name="rows"></param>
+        static member validateStringSeqs(headers:#seq<string>, rows:#seq<#seq<string>>) =
+
+            let columns = 
+                Seq.append [headers] rows 
+                |> Seq.transpose
+
+            let columnsArray = 
+                columns 
+                |> Seq.toArray
+                |> Array.map (Seq.toArray)
+
+            let groupedColumns = columnsArray |> ArcTable.groupColumnsByHeader
```

Comment:
If you're going to validate the table column-wise anyways. Why not have a function which valdiates a single column and call that one sequentially here? This would increase readability I think.

Reply:
The problem is, I cannot really validate them column wise but based on the building blocks.
I have no Idea how you would validate a single column because a free text column can have any kind of information.
The only thing that could be validated whether the value in a term column is right or wront or am I overlooking something?

Reply:
Sry, single-column is not the correct term. I mean a column-group or single composite column.

Reply:
The problem is, that the grouping method isn't working properly due to the false namings of the error and that makes a method/ function quite useless or not?